source/beamline/AMTemperatureMonitors.h

```diff
@@ -0,0 +1,43 @@
+#ifndef AMTEMPERATUREMONITORS_H
+#define AMTEMPERATUREMONITORS_H
+
+#include "beamline/AMConnectedControl.h"
+#include "beamline/AMTemperatureMonitorsStatus.h"
+
+class AMTemperatureMonitors : public AMConnectedControl
```

Comment:
I'm almost confused this class with `AMTemperatureMonitor`. Therefore, I'm wondering whether we could rename this class to `AMTemparatureMonitorGroup` or `AMTemparatureMonitorSet`, or sth alike?